src/components/SearchBar.js

```diff
@@ -0,0 +1,16 @@
+import '../styles/SearchBar.css';
+import iconSearch from '../assets/icons/search.svg';
+
+function SearchBar() {
+
+  return (
+    <div className="searchbar">
+      <div className="searchbar-content">
+        <img className="searchbar-icon" src={iconSearch} alt="링크를 검색하기." />
+        <input className="searchbar-input" type="text" placeholder="링크를 검색해 보세요."></input>
+      </div>
+    </div>
```

Comment:
가장 바깥쪽 `div`의 역할이 딱히 의미가 없는 것 같습니다. 마크업은 최대한 간결하게 해주세요
`searchbar-content` 자리에 `searchbar` 클래스를 주고 가장 바깥 `searchbar`를 없애도 동일한 UI이 나오네요 

```suggestion
<div className="searchbar">
      <img className="searchbar-icon" src={iconSearch} alt="링크를 검색하기." />
      <input className="searchbar-input" type="text" placeholder="링크를 검색해 보세요."></input>
    </div>
```

Reply:
2e2f79993e4c3c3f2de4c227fd637bdf5912d8a7 수정했습니다 ~!